geojson/geojson_shapes_util.go

```diff
@@ -29,11 +29,12 @@ var jsoniter = jsoniterator.ConfigCompatibleWithStandardLibrary
 
 // FilterGeoShapesOnRelation extracts the shapes in the document, apply
 // the `relation` filter and confirms whether the shape in the document
-//  satisfies the given relation.
+//
+//	satisfies the given relation.
```

Comment:
Undo this indent, `go fmt ./...` must've changed this because of the unnecessary white space before `satisfies` in line 32 above.